typescript/packages/plugins/ionic/src/parameters.ts

```diff
@@ -0,0 +1,50 @@
+import { createToolParameters } from "@goat-sdk/core";
+import { z } from "zod";
+
+export class SupplyAssetParameters extends createToolParameters(
+  z.object({
+    poolId: z.string().describe("The ID of the pool to supply to"),
```

Comment:
What is a Pool id? and where do i get the pool id from in Ionic?

Reply:
Is pool id something that changes often? If not, I would set it as optional and have a default pool. Seems like default is main/1? I haven't seen other pools, do you have examples?